### PR TITLE
Number7

### PR DESCRIPTION
**Pull request by samnawi**

### CODE REVIEW

**Comment by adisadit** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # scrum-demo
 
+This project is the most awesome open source project ever.https://github.com/AIFDR/scrum-demo.githttps://github.com/AIFDR/scrum-demo.git
+
+
+bla bla la
```
Comment (harrymahar) [inline README.md:6]: **_You Fool**_

**Comment by harrymahar** [comment]
Iwan, LVGTM, Mantap Gan :+1:

File: README.md (hunk 1 above)
Comment (samnawi) [inline README.md:6]: This line is specially here for version 4

**Pull request merged**